Dockerfile

```diff
@@ -203,6 +203,7 @@ RUN chmod +x /usr/sbin/tini
 
 # Add startup script
 ADD scripts/start-mspass.sh /usr/sbin/start-mspass.sh
+ADD scripts/start-mspass-tacc.sh /usr/sbin/start-mspass-tacc.sh
```

Comment:
We don't necessarily want to include that script here in the main container image.